fwdctl/src/dump.rs

```diff
@@ -1,15 +1,15 @@
 // Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE.md for licensing terms.
 
-use std::borrow::Cow;
-
 use clap::Args;
 use firewood::{
     db::{Db, DbConfig, WalConfig},
+    merkle::Key,
     v2::api::{self, Db as _},
 };
 use futures_util::StreamExt;
 use log;
+use std::borrow::Cow;
```

Comment:
nit: why was this suddenly needed?

Reply:
It used to be imported above but it seems that the format-on-save moved this line for some reason